### DISCUSSION TITLE
Fetch if aggregate value of the returned table rows by the computed field satisfies a condition

### DISCUSSION THREAD

**Discussion by tienpvse1** [hasura/graphql-engine#8865] [Question] [answered]
![image](https://user-images.githubusercontent.com/86212960/183275941-666928ce-1779-4fc1-9978-2c599e4c126f.png)
 
Currently in the document, this feature seems to exist and pretty helpful for me, however , I've tried many times to create computed fields, this filter using aggregate value feature doesn't exist 
Link to the document: https://hasura.io/docs/latest/queries/postgres/query-filters/#fetch-if-aggregate-value-of-the-returned-table-rows-by-the-computed-field-satisfies-a-condition

**Comment by ajohnson1200** [comment]
Thanks for the post!  Could you add the post the schema, the SQL function you used, and the GraphQL query you used to reproduce this?

**Comment by robertjdominguez** [comment, marked as answer]
I'm converting this to a discussion, @tienpvse1 👍 

If you have a follow-up question or can provide the requested information above, we'll see what we can do to help!
